package/MDAnalysis/analysis/hydrogenbonds/hbond_analysis.py

```diff
@@ -720,6 +741,8 @@ def _single_frame(self):
 
         # find D and A within cutoff distance of one another
         # min_cutoff = 1.0 as an atom cannot form a hydrogen bond with itself
+        print(f"D POS: {self._donors.positions}")
```

Comment:
You can leave the debugging as it is but please put it with a logger. Check https://github.com/MDAnalysis/mdanalysis/blob/804b6073fdfaee0c5be1e2be659c2280ee4d0cda/package/MDAnalysis/analysis/align.py#L221
as an example

Reply:
Actually, if it's purely debugging output that you used during development, remove it. Even if you include it at level DEBUG, it might slow down people who need to enable a debug logger for other pieces of code.

Reply:
> You can leave the debugging as it is but please put it with a logger. 

I am already planning to remove all these debugging and commented regions for the final code I commit, I just wanted to check my solution was appropriate before doing so (incase I still needed it).